code/modules/antagonists/nuclear_operative/nuclear_operative.dm

```diff
@@ -99,6 +101,37 @@
 
 		. = ..()
 
+	get_statistics()
+		var/list/purchases = list()
+		// Add items purchased from the nukies weapon vendor
+		for (var/datum/materiel/purchased_item as anything in src.purchased_items)
```

Comment:
Why `/datum/materiel`?

Reply:
datum/materiel is what weapon_vendor.dm uses, and I believe everything added to purchased_items should be a materiel. (See lines 220 or 78 in weapon_vendor) Maybe something else would be better? Unsure